logger.go

```diff
@@ -4,9 +4,10 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"sync"
 )
 
-var lflags = log.Ldate | log.Lmicroseconds | log.Lshortfile
+var std = &logger{l: DefaultLogger(), lvl: LogLevelInfo}
```

Comment:
Oh sorry when I switched from github.split view to normal diff, it looks much better...